packages/autocomplete-core/src/getPropGetters.ts

```diff
@@ -219,6 +220,25 @@ export function getPropGetters<
       maxLength,
       type: 'search',
       onChange: (event) => {
+        const value = (
+          (event as unknown as Event).currentTarget as HTMLInputElement
+        ).value;
+
+        if (getNativeEvent(event as unknown as InputEvent).isComposing) {
```

Comment:
I see the compatibility layer in core makes sense here, but I wonder if it actually should be part of the react layer? I guess once that's not just an example anymore?)

Reply:
You mean as user land code as there's no React layer ?
I think it's fine to just have a ternary checking whether there's `nativeEvent` or not, doesn't add much overhead

Reply:
Yes, I was thinking as user land code, but it's indeed easy to forget